src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -67,14 +67,14 @@ public class LinkTagHelper : TagHelper
         private enum Mode
         {
             /// <summary>
-            /// Rendering a fallback block if primary stylesheet fails to load. Will also do globbing if the appropriate
-            /// properties are set.
+            /// Just performing file globbing search for the href, rendering a separate &lt;link&gt; for each match.
             /// </summary>
-            Fallback,
+            GlobbedHref = 1,
             /// <summary>
-            /// Just performing file globbing search for the href, rendering a separate &lt;link&gt; for each match.
+            /// Rendering a fallback block if primary stylesheet fails to load. Will also do globbing for both the
+            /// primary and fallback hrefs if the appropriate properties are set.
             /// </summary>
-            GlobbedHref
+            Fallback = 2,
```

Comment:
I provided explicit values to clearly indicate that I'm using the enum in an ordered fashion. I can make them 0 and 1, that's just me being dumb.